src/Integration.UnitTests/Binding/NuGetHelperTests.cs

```diff
@@ -37,33 +37,34 @@ public class NuGetHelperTests
         [TestMethod]
         public void NuGetHelper_ArgChecks()
         {
-            Exceptions.Expect<ArgumentNullException>(() => NuGetHelper.TryInstallPackage(null, new ProjectMock("123"), "123"));
-            Exceptions.Expect<ArgumentNullException>(() => NuGetHelper.TryInstallPackage(CreateServiceProvider(), null, "123"));
-            Exceptions.Expect<ArgumentNullException>(() => NuGetHelper.TryInstallPackage(CreateServiceProvider(), new ProjectMock("123"), null));
+            var loggerMock = new Mock<ILogger>().Object;
+
+            Exceptions.Expect<ArgumentNullException>(() => NuGetHelper.TryInstallPackage(null, loggerMock, new ProjectMock("123"), "123"));
+            Exceptions.Expect<ArgumentNullException>(() => NuGetHelper.TryInstallPackage(CreateServiceProvider(), null, new ProjectMock("123"), "123"));
+            Exceptions.Expect<ArgumentNullException>(() => NuGetHelper.TryInstallPackage(CreateServiceProvider(), loggerMock, null, "123"));
+            Exceptions.Expect<ArgumentNullException>(() => NuGetHelper.TryInstallPackage(CreateServiceProvider(), loggerMock, new ProjectMock("123"), null));
         }
 
         [TestMethod]
         public void NuGetHelper_HandleFailures()
         {
             // Arrange
-            ConfigurableServiceProvider sp = CreateServiceProvider();
-            var outputWindow = new ConfigurableVsOutputWindow();
-            var outputPane = outputWindow.GetOrCreateSonarLintPane();
-            sp.RegisterService(typeof(SVsOutputWindow), outputWindow);
+            var sp = CreateServiceProvider();
+            var testLogger = new TestLogger();
```

Comment:
Minor refactoring of the _NuGetHelper_ so that it accepts an _ILogger_ in the constructor.
Previously, it was using a static helper method to write to the VS Output pane.